index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>My portfolio</h1>
+    </section>
+
+    <section id="projects">
+      <h1 class="project-tile">My project</h1>
+      <div>
+        <a id="profile-link" href="https://github.com/wassimchakib/html_css_dami_wassim"
+      target = "_blank"> 
```

Comment:
Good job so far, Please try to avoid space between attributes and values as much as you can

Reply:
- [OPTIONAL] Nice one @wassimchakib, but try as much as you can to include `WHY` the student is asked to make of a suggestion as this would make it clearer for the student why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where where more information can got for  more clarification.